robots/pr-creator/updater/updater.go

```diff
@@ -67,17 +67,16 @@ func UpdatePR(org, repo, title, body, matchTitle string, gc updateClient) (*int,
 	return &n, nil
 }
 
-func EnsurePR(org, repo, title, body, source, branch, matchTitle string, gc ensureClient) (*int, error) {
-	return EnsurePRWithLabels(org, repo, title, body, source, branch, matchTitle, gc, nil)
+func EnsurePR(org, repo, title, body, source, branch, matchTitle string, allowMods bool, gc ensureClient) (*int, error) {
+	return EnsurePRWithLabels(org, repo, title, body, source, branch, matchTitle, allowMods, gc, nil)
 }
 
-func EnsurePRWithLabels(org, repo, title, body, source, branch, matchTitle string, gc ensureClient, labels []string) (*int, error) {
+func EnsurePRWithLabels(org, repo, title, body, source, branch, matchTitle string, allowMods bool, gc ensureClient, labels []string) (*int, error) {
 	n, err := UpdatePR(org, repo, title, body, matchTitle, gc)
 	if err != nil {
 		return nil, fmt.Errorf("update error: %v", err)
 	}
 	if n == nil {
-		allowMods := true
```

Comment:
might be useful to export values for people to use for readability, aka
```golang
const (
  AllowMods = true
  PreventMods = false
)
```

Reply:
Makes sense. I will add constants for these,